python/pyspark/sql/session.py

```diff
@@ -156,7 +156,7 @@ def getOrCreate(self):
             default.
 
             >>> s1 = SparkSession.builder.config("k1", "v1").getOrCreate()
-            >>> s1.conf.get("k1") == s1.sparkContext.getConf().get("k1") == "v1"
+            >>> s1.conf.get("k1") == "v1"
```

Comment:
@ueshin Could we also update the migration guide about this change?

Reply:
In that case, we might have to put the behaviour changes by https://github.com/apache/spark/pull/18536 together to the migration guide as well.

Reply:
We can do it together.

Reply:
Submitted a pr to update the migration guide #22682.